src/sniffer/ErrorHandler.c

```diff
@@ -0,0 +1,51 @@
+
+#include "ErrorHandler.h"
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "InterfaceHandler.h"
+#include "IOHandler.h"
+#include "Sniffer.h"
+
+char ErrorHandler_pcapErrBuff[PCAP_ERRBUF_SIZE];
+
+static void CleanUpAndExit() {
+    Sniffer_CleanUp();
+    InterfaceHandler_CleanUp();
+    IOHandler_CleanUp();
+    exit(1);
+}
+
+void ErrorHandler_DisplayPcapErrorAndExit(char *error_message, bool printErrBuff) {
+    printf("-=-=-=-=-=-=-= PCAP ERROR! =-=-=-=-=-=-=-\n");
+    printf("%s\n", error_message);
+    if (printErrBuff == true) {
```

Comment:
Correct me if I am wrong, but I am guessing the purpose of this is to handle those pcap functions that won't write to the `errbuff` and we would display a wrong error message in case we printed it, right?